src/web/shared/Styles.ts

```diff
@@ -377,18 +379,18 @@ export function getWaypointCircleStyle(feature: Feature<Point>) {
 
     //The reason we need to divide by the cosine of the 
     // latitude is because the map is using a Mercator projection, (with units in meters at the equator)
-    const latitudeCoefficient = Math.max(Math.cos((goal.location.lat ?? 0) * DEG), 0.001)
+    const latitudeCoefficient = Math.max(Math.cos((goal.location?.lat ?? 0) * DEG), 0.001)
     const captureRadius = 5.0 / latitudeCoefficient // meters, MOOS configuration from templates/bot/bot.bhv.in
-    const centerCoordinate = feature.getGeometry().getCoordinates()
+    const centerCoordinate = feature.getGeometry()!.getCoordinates()
     const colorName = getGoalColor(isActive, isSelected, canEdit)
     const colorMain = colorNameToHex(colorName) ?? colorName
-    const colorBorder = colorNameToHex('black')
+    const colorBorder = '#000'
 
     function getCircleStyle(center: Coordinate, radius: number, color: string, lineWidth: number, addInnerGradientColor: boolean) {
```

Comment:
Can we add the a comment before this function like the driftspeedtobinindex function  /***/

Reply:
https://github.com/jaiarobotics/jaiabot/blob/1.y/src/doc/markdown/page_110_coding_standards.md